nestkernel/simulation_manager.cpp

```diff
@@ -883,9 +948,17 @@ nest::SimulationManager::update_()
         }
 
       } // of if(wfr_is_used)
-      // end of preliminary update
+        // end of preliminary update
 
+#ifdef TIMER_DETAILED
+#pragma omp barrier
```

Comment:
Same as above.

Reply:
Yes, it's not frequently called. Keep this barrier.